gradle.properties

```diff
@@ -42,4 +46,9 @@ POM_SCM_DEV_CONNECTION=scm:git:ssh://git@github.com/snappdevelopment/JsonTree.gi
 
 POM_DEVELOPER_ID=snappdevelopment
 POM_DEVELOPER_NAME=Sebastian Neubauer
-POM_DEVELOPER_URL=https://github.com/snappdevelopment/
\ No newline at end of file
+POM_DEVELOPER_URL=https://github.com/snappdevelopment/
+
+# wasm falgs
+org.jetbrains.compose.experimental.jscanvas.enabled=true
+org.jetbrains.compose.experimental.macos.enabled=true
```

Comment:
Do we need this flag? From a Google search it seems like this is for an experimental native macOS target. I haven't tried that yet. I don't think we need it for now.

Reply:
I think can remove np

Reply:
Let's remove it for now